drivers/staging/vc04_services/bcm2835-audio/bcm2835.c

```diff
@@ -79,7 +79,11 @@ static int bcm2835_audio_alsa_newpcm(struct bcm2835_chip *chip,
 	if (err)
 		return err;
 
-	err = snd_bcm2835_new_pcm(chip, "bcm2835 IEC958/HDMI", 1, 0, 1, true);
+	err = snd_bcm2835_new_pcm(chip, "bcm2835 IEC958/HDMI0", 1, AUDIO_DEST_HDMI0, 1, true);
```

Comment:
Are there any consequences of this name change, e.g. in ALSA configuration files, the LXPanel volume plugin etc?

Reply:
Possibly, but I am not sure where or when those particular items are used - not by us AFAIK.